pages/utils/sum.ts

```diff
@@ -0,0 +1,8 @@
+export function sum(a: number, b: number, format: "html" | "text" = "html") {
```

Comment:
Maybe it's best to add a comment for this function e.g jsdocs to have descriptions for the functionality of this function

Reply:
Yes. lets do that. Can you give me reference to JSDocs and how we can do that?